yarn.lock

```diff
@@ -6903,9 +6903,9 @@ unc-path-regex@^0.1.2:
   resolved "https://registry.yarnpkg.com/unc-path-regex/-/unc-path-regex-0.1.2.tgz#e73dd3d7b0d7c5ed86fbac6b0ae7d8c6a69d50fa"
   integrity sha1-5z3T17DXxe2G+6xrCufYxqadUPo=
 
-"undici@github:mcollina/undici":
-  version "0.5.0"
-  resolved "https://codeload.github.com/mcollina/undici/tar.gz/ed672a39ee63f4c5fa13d0c826a0d47edf77e9fa"
+undici@mcollina/undici:
```

Comment:
🤷‍♂️

Reply:
Yeah saw that recently too, I have no idea where it comes from